src/draw/vg_lite/lv_vg_lite_math.c

```diff
@@ -39,12 +39,12 @@
 
 float math_fast_inv_sqrtf(float number)
 {
-    int32_t i;
-    float x2, y;
     const float threehalfs = 1.5f;
 
-    x2 = number * 0.5f;
-    y = number;
```

Comment:
I don't see any problem with the original code, all variables are assigned before being read. Can you provide the compiler name, version number and original warning message?

Reply:
@FASTSHIFT the original code instance y ans x2 without proper value, even though they get assigned at first opportunity this is sufficient to raise the warnings on modern GCC compilers of uninitialized variable, so if a project treats all warnings as errors this gets promoted to a build failure.

Assigning them during declaration is sufficient to ditches away the message without making any changes on the host environment.

Reply:
OK